tests/test_utils.py

```diff
@@ -0,0 +1,129 @@
+import warnings
```

Comment:
Please add the tests to tox.ini

Reply:
Is this not handled by?
https://github.com/RedHatQE/openshift-python-wrapper/blob/master/tox.ini#L24

Reply:
No, we run tox -e code-check
https://github.com/RedHatQE/openshift-python-wrapper/blob/master/.github/workflows/code-check.yml#L33

you should create unittests section and add it to the workflow file
tox -e unittests